pkg/api/util/util.go

```diff
@@ -110,3 +110,14 @@ func ValidateDCARequest(w http.ResponseWriter, r *http.Request) error {
 
 	return err
 }
+
+// IsForbidden returns if a the cluster check runner server is allowed to listen on a given ip
+func IsForbidden(ip string) bool {
```

Comment:
The purpose of this function is be a safeguard against the default `""` indeed and to make it clear to users that the server is not meant to listen on wildcard IP addresses, but it still not a security function. It's an additional step to avoid a potential unnecessarily permissive config (using wildcard IP)

@DataDog/agent-core we would appreciate your suggestions here as well

Reply:
If it's realistic to implement a validation that handles all these edge cases, let's do so.

If not, I think the function docs _and_ its usage should make it clear that this is not a real security feature that guarantees anything, but just a non-secure helper that helps users avoid mistakenly setting an IP that's too permissive.